avtools/avconv.c

```diff
@@ -1133,15 +1198,26 @@ static void do_streamcopy(InputStream *ist, OutputStream *ost, const AVPacket *p
         return;
     }
 
+    MUTEX_LOCK(&of->data_lock);
+    start_time = (of->start_time == AV_NOPTS_VALUE) ? 0 : of->start_time;
+    MUTEX_UNLOCK(&of->data_lock);
+    MUTEX_LOCK(&ost->data_lock);
+
+    ost_tb_start_time = av_rescale_q(start_time, AV_TIME_BASE_Q, ost->mux_timebase);
+
     av_init_packet(&opkt);
```

Comment:
I sent a patch to move the `av_init_packet` before line 1196 already, maybe this whole block should be moved up as well.